README.md

```diff
@@ -117,13 +117,16 @@ Submit issues at: https://github.com/ocp-power-automation/openshift-install-powe
 ```
 
 ## Prerequisites
-The script assumes PowerVS prerequisites for OpenShift are already in place.
+**The script assumes PowerVS prerequisites for OpenShift are already in place.**<br>
+In case you missed, here are the links to the prerequisites:
+1. For setting up the network in Power Virtual Server service, please refer this [link](https://developer.ibm.com/tutorials/setting-up-powervs-for-ocp/)
+2. For instructions on how to prepare RHEL & RHCOS OVA images, please refer this [link](https://developer.ibm.com/tutorials/preparing-ova-for-openshift-on-cloud/)
```

Comment:
How about just linking to `https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/docs/ocp_prereqs_powervs.md` ?
The blogs can go stale very soon and I think its better to the link the github readme ..

Reply:
@bpradipt fair point.. update the patch to reflect the same.

Reply:
thanks @dpkshetty. 
The patch lgtm. However before merge, can you please squash the commits into a single one, add SOB and re-push.